bitnami/wordpress/README.md

```diff
@@ -626,4 +626,4 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
```

Comment:
Now that we saw that the action works, could you undo this change?

Reply:
Yes, done!